DependencyInjection/Compiler/ReplaceUserCheckerCompilerPass.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Modera\SecurityBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author    Konstantin Myakshin <koc-dp@yandex.ru>
```

Comment:
IHO name of this class says that it is compiler pass which replaces user checker. Also there is inline comment on 27th line.